internal/cli/atlas/livemigrations/validation/create.go

```diff
@@ -0,0 +1,121 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validation
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/openlyinc/pointy"
+	"github.com/spf13/cobra"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+type CreateOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	destinationClusterName      string
+	destinationDropEnabled      bool
+	migrationHosts              []string
+	sourceCACertificatePath     string
+	sourceClusterName           string
+	sourceProjectID             string
+	sourceSSL                   bool
+	sourceManagedAuthentication bool
+	sourceUserName              string
+	sourcePassword              string
+	store                       store.LiveMigrationValidationsCreator
+}
+
+func (opts *CreateOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(store.AuthenticatedPreset(config.Default()))
+	return err
+}
+
+var createTemplate = `ID	PROJECT ID	SOURCE PROJECT ID	STATUS
+{{.ID}}	{{.GroupID}}	{{.SourceGroupID}}	{{.Status}}`
+
+func (opts *CreateOpts) Run() error {
+	createRequest := opts.newValidationCreateRequest()
+
+	r, err := opts.store.CreateValidation(opts.ConfigProjectID(), createRequest)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *CreateOpts) newValidationCreateRequest() *mongodbatlas.LiveMigration {
+	return &mongodbatlas.LiveMigration{
+		Source: &mongodbatlas.Source{
+			ClusterName:           opts.sourceClusterName,
+			GroupID:               opts.sourceProjectID,
+			Username:              opts.sourceUserName,
+			Password:              opts.sourcePassword,
+			SSL:                   pointy.Bool(opts.sourceSSL),
+			CACertificatePath:     opts.sourceCACertificatePath,
+			ManagedAuthentication: pointy.Bool(opts.sourceManagedAuthentication),
+		},
+		Destination: &mongodbatlas.Destination{
+			ClusterName: opts.destinationClusterName,
+			GroupID:     opts.ConfigProjectID(),
+		},
+		MigrationHosts: opts.migrationHosts,
+		DropEnabled:    pointy.Bool(opts.destinationDropEnabled),
+	}
+}
+
+// mongocli atlas liveMigrations|lm validation create --clusterName clusterName --migrationHosts hosts --sourceClusterName clusterName --sourceProjectId projectId [--sourceSSL] [--sourceCACertificatePath path] [--sourceManagedAuthentication] [--sourceUserName userName] [--sourcePassword password] [--drop] [--projectId projectId].
+func CreateBuilder() *cobra.Command {
+	opts := &CreateOpts{}
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create one new validation request.",
+		Long:  "Your API Key must have the Organization Owner role to successfully run this command.",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(
+				opts.ValidateOrgID,
```

Comment:
[nit] since you are using `--projectId` and `ConfigProjectID()` you should add 
```suggestion
				opts.ValidateOrgID,
				opts.ValidateProjectID,
```